src/parse_expr.jl

```diff
@@ -4,10 +4,9 @@
 #  file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 function try_parse_idx_set(arg::Expr)
-    is_julia_v1 = @static (VERSION >= v"1.0-" ? true : false)
```

Comment:
Is there a different behavior between 0.7 and 1.0? The VERSION check might still be needed.

Reply:
Indeed, I went too fast on this one. This is due to a change that occured between v0.7 and v1.0: https://github.com/JuliaOpt/JuMP.jl/pull/1428